integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/playground/SourceCodeSnippetPlaygroundTests.java

```diff
@@ -0,0 +1,36 @@
+package eu.tsystems.mms.tic.testframework.playground;
+
+import eu.tsystems.mms.tic.testframework.exceptions.PageFactoryException;
+import eu.tsystems.mms.tic.testframework.test.utils.AbstractSourceCodeTests;
+import org.testng.annotations.Test;
+
+/**
+ * Created on 2022-10-07
+ *
+ * @author mgn
+ */
+public class SourceCodeSnippetPlaygroundTests extends AbstractSourceCodeTests {
+
+    // These tests need the following property:
+    // tt.report.source.exclusion.regex=<nothing>
+    @Test(expectedExceptions = AssertionError.class)
+    public void T05_default_source_code_exclusion_pretest() {
+        this.doSimpleFail();
```

Comment:
Huh. I'd call this test paranoid even for my standards. I mean, what does it mean, if `Assert.fail` does not throw anymore?

... FWIW: Even though the comment claims otherwise, this test happily succeeds with `tt.report.source.exclusion.regex=AbstractSourceCodeTests`.

Reply:
Yes, only that testcase is independend from the property but the whole test class not.

These tests have to migrated to ReportNG tests. At the moment they are still in development. If this PR is accepted I can merge that to the correspondig branch.